code/modules/hippie/items/stapler.dm

```diff
@@ -174,4 +174,75 @@
 			else
 				user << "<span class='notice'>There is already a butt in \the [src]!</span>"
 		else
-			user << "<span class='notice'>There is already a paper in \the [src]!</span>"
\ No newline at end of file
+			user << "<span class='notice'>There is already a paper in \the [src]!</span>"
+
+//coffin nailing for stapler instead of welding
+/obj/structure/closet/coffin/update_icon()
```

Comment:
Used to be in there, But i kinda moved it in here to be a parto of hippie module and stapler.
After all it relies on our own stapler code.